contracts/JBFundingCycleStore.sol

```diff
@@ -465,9 +505,19 @@ contract JBFundingCycleStore is JBControllerUtility, IJBFundingCycleStore {
     // Get the necessary properties for the base funding cycle.
     JBFundingCycle memory _baseFundingCycle = _getStructFor(_projectId, _fundingCycle.basedOn);
 
+    // Find the base cycle that is not still queued.
+    while (_baseFundingCycle.start > block.timestamp) {
```

Comment:
do-while loop?

Reply:
need a while (true) loop with a conditional here. i made the change.